make.js

```diff
@@ -192,9 +190,8 @@ CLI.build = function(/** @type {{ node: string; task: string }} */ argv)
     CLI.serverBuild(argv);
 }
 
-CLI.serverBuild = function(/** @type {{ node: string; task: string }} */ argv) {
+CLI.serverBuild = function(/** @type {{ task: string }} */ argv) {
```

Comment:
It would be good to trigger these changes for all tasks to ensure build works with these changes